omero/sysadmins/server-performance.rst

```diff
@@ -186,7 +186,7 @@ to expose further properties.
 
 With :ref:`further configuration <jmx_configuration>`, |JMX| properties
 can also be accessed remotely which can be very useful for monitoring
-your server with `Check_MK`_, `Nagios`_, `Zenoss`_, or similar. However,
```

Comment:
Is "Check MK" no longer the official name?

Reply:
it is now Checkmk the official name

Reply:
Cheers.